packages/cli/commands/init.js

```diff
@@ -109,7 +109,7 @@ exports.handler = async options => {
     const path = getConfigPath();
 
     logger.success(
-      `The config file "${path}" was created using your personal access key for account ${name ||
+      `The config file "${path}" was created using ${authType} for account ${name ||
```

Comment:
Should we surround with double quotes since the `authType` is a token not a friendly name?